multi_import/fields.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import relations
+
+from multi_import.utils import normalize_string
+
+__all__ = [
+    'FieldHelper',
+]
+
+
+class FieldHelper(object):
+    list_separator = ';'
+
+    def to_string_representation(self, field, value):
+        if hasattr(field, 'to_string_representation'):
```

Comment:
I kind of feel like the Field should not be responsible for having a "to_string_representation". It should just have a to_representation. It should be the consumers job to use that representation however they want to.